src/connectors/numeric-menu/connectNumericMenu.js

```diff
@@ -271,11 +271,11 @@ function refine(state, attribute, items, facetValue) {
   const currentRefinements = resolvedState.getNumericRefinements(attribute);
```

Comment:
I noticed something slighly weird with this widget.
![2019-09-05 12 48 04](https://user-images.githubusercontent.com/2982512/64335710-af755300-cfdb-11e9-85ae-5618b6f84f6b.gif)

A numeric menu added to the first subindex also filters on the second one.
Let me know if you have trouble reproducing it. I would created a test in connectNumericMenu directly but it's too unit-testy to add a multi-index test in there

Reply:
this is expected, refinements are inherited; unless they  contain that widget in their own state too.